ooniprobe/View/Settings/SettingsTableViewController.m

```diff
@@ -113,9 +116,6 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         else {
             cell = [tableView dequeueReusableCellWithIdentifier:@"Cell" forIndexPath:indexPath];
         }
-        if ([[TestUtility getTestOptionTypes] containsObject:current]){
-            cell.imageView.image = [UIImage imageNamed:current];
-        }
```

Comment:
Can you help me understand what was the purpose of this check and of the code executed if true?

Reply:
Got it